### PR TITLE
fix(replies): change avatar color to correct one

### PR DESCRIPTION
**Pull request by AlexZakablukov**
**What this PR does** 📖
change avatar color to correct one

**Which issue(s) this PR fixes** 🔨
[AP-1411](https://satellite-im.atlassian.net/browse/AP-1411)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by phillsatellite** [comment]
Tested: Noticed on Safari if user has a  profile picture, the profile picture will not appear in they're reply 
Screen Shot 2022-04-14 at 1 13 55 PM

**Comment by AlexZakablukov** [comment]
fixed

**Review by phillsatellite — APPROVED** [review]

**Review by Jekrimo — APPROVED** [review]

File: components/views/chat/message/reply/Item/Item.vue [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      type: Function,
+      default: () => {},
+    },
+    emojiReaction: {
+      type: Function,
+      default: (): void => {},
+    },
+    showQuickProfile: {
+      type: Function,
+      default: (): void => {},
```
Comment (molimauro) [inline components/views/chat/message/reply/Item/Item.vue:41]: @AlexZakablukov do you think we can emit here instead?

Comment (AlexZakablukov) [reply components/views/chat/message/reply/Item/Item.vue]: done

**Review by molimauro — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**